R/ml_kmeans.R

```diff
@@ -54,19 +56,33 @@ ml_kmeans <- function(x,
   # extract cluster centers
   kmmCenters <- invoke(fit, "clusterCenters")
 
+  # compute cost for k-means
+  if (compute.cost)
```

Comment:
No problem : ) this is the functionality that I need so it's vital for me.
Talking about unit tests, I think they are very simple right now and I
think I'll come up with a more complex one.

2016-08-17 18:49 GMT+02:00 Kevin Ushey <notifications@github.com>:

> In R/ml_kmeans.R
> <https://github.com/rstudio/sparklyr/pull/173#discussion_r75161015>:
>
> > @@ -54,19 +56,33 @@ ml_kmeans <- function(x,
> >    # extract cluster centers
> >    kmmCenters <- invoke(fit, "clusterCenters")
> >
> > +  # compute cost for k-means
> > +  if (compute.cost)
>
> Let's default to TRUE and allow users to opt out if it's too expensive
> for their particular case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparklyr/pull/173/files/664b588fe99160f592c8892ad86ab6d363dbf297#r75161015>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGdazurSriRzfnVi9_mz420gacY5SbuIks5qgzuCgaJpZM4JlWxk>
> .
>